x-pack/plugins/log_explorer/public/components/flyout_detail/sub_components/highlight_field.tsx

```diff
@@ -125,3 +140,16 @@ export function HighlightField({
     </EuiFlexGroup>
   ) : null;
 }
+
+// POC code just to try visualize how we can access info about a field name
+const useFieldDescription = (fieldName: string) => {
+  const [field, setField] = useState(null);
+
+  useEffect(() => {
+    fetch(`http://localhost:5000/explain?name=${encodeURIComponent(fieldName)}`)
```

Comment:
In case the data has a dataset in, it can also be added to the query to get even more details.